style.css

```diff
@@ -112,3 +112,12 @@ section h2 {
   margin-left: -4px;
   top: 10px;
 }
+
+.parallax {
```

Comment:
You can put background image under #intro because img src is only relative to intro. Also you cam put entire code under intro or keep .parallax for re using with other section.

Reply:
Thanks for the review. 
I've put the entire parallax code under #intro.